### PR TITLE
Implement AVIF Image Format Support 

### PR DESCRIPTION
**Pull request by vamsii777**
This PR introduces support for the AVIF image format in the SwiftGD library. The addition of AVIF format capabilities enhances the library's versatility, aligning it with modern image processing standards.

#### Changes
- **AVIF Formatter Implementation**: Added `AVIFFormatter` structures, conforming to `LibGdImportableFormatter` and `LibGdExportableFormatter`, to handle AVIF image import and export.
- **Enum Extensions**: Updated `ImportableFormat` and `ExportableFormat` enums to include the AVIF format, allowing for seamless integration of AVIF handling in the existing SwiftGD workflow.
- **Testing and Validation**: Conducted tests of AVIF images to ensure proper functionality and integration with the existing codebase.

#### Motivation
AVIF, as a modern and efficient image format, is increasingly being adopted for its superior compression and image quality. Integrating AVIF support into SwiftGD ensures that the library remains up-to-date with current image processing trends and technologies.

#### Impact
This addition provides SwiftGD users the ability to work with AVIF images, extending the library's applicability to a broader range of projects and use cases. It also ensures compatibility with web and application development standards that are increasingly incorporating AVIF.

Looking forward to your feedback and suggestions for any further improvements.

### CODE REVIEW

File: Sources/SwiftGD/Format.swift [hunk 1]
```diff
@@ -116,7 +116,7 @@ extension LibGdParametrizableExportFormatter {
 private struct BMPFormatter: LibGdParametrizableFormatter {
     /// The parameters to apply on exports
     fileprivate var exportParameters: Int32
-
+
```
Comment (mcritz) [inline Sources/SwiftGD/Format.swift:119]: Nit: don’t change line formatting

File: Sources/SwiftGD/Format.swift [hunk 2]
```diff
@@ -50,7 +50,7 @@ private protocol LibGdExportableFormatter: ExportableFormatter {
 private protocol LibGdParametrizableExportFormatter: ExportableFormatter {
     /// The parameters to apply on exports
     var exportParameters: Int32 { get }
-
+
```
Comment (mcritz) [inline Sources/SwiftGD/Format.swift:53]: Nit: don’t change line formatting

File: Sources/SwiftGD/Format.swift [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #if os(Linux)
-    import Glibc
+import Glibc
```
Comment (mcritz) [inline Sources/SwiftGD/Format.swift:2]: Nit: don’t change line formatting

File: Tests/SwiftGDTests/ImageTests.swift [hunk 4]
```diff
@@ -30,27 +30,53 @@ class TestImage: XCTestCase {
...
         guard let image = Image(width: 640, height: 480) else {
             throw Error.invalidImage(reason: "Could not create image")
         }
         let renderBounds = image.renderText("Hello, World", from: .zero, fontList: [], color: .white, size: 18.0)
-
+        
         XCTAssertTrue(try isEmptyBounds(for: renderBounds), "Empty fontLists return tuple of zero-value Points")
     }
+    
+    func testCreateAndExportAVIFImage() throws {
```
Comment (mcritz) [inline Tests/SwiftGDTests/ImageTests.swift:55]: Good!

File: Tests/SwiftGDTests/ImageTests.swift [hunk 5]
```diff
@@ -16,7 +16,7 @@ class TestImage: XCTestCase {
         "Noto Sans",
         "SSTPro-Roman"
     ]
-
+
```
Comment (mcritz) [inline Tests/SwiftGDTests/ImageTests.swift:19]: Nit: don’t change line formatting

**Review by mcritz — CHANGES_REQUESTED** [review]
IMHO, this is a very helpful PR. Thanks for adding tests. Personally, I avoid breaking formatting when contributing to others’ projects.

File: Sources/SwiftGD/Format.swift (hunk 1 above)
Comment (vamsii777) [reply Sources/SwiftGD/Format.swift]: Yes, I apologize; I'm used to using Xcode's `Ctrl+I` key. I've corrected all code format please check.

File: Tests/SwiftGDTests/ImageTests.swift (hunk 4 above)
Comment (vamsii777) [reply Tests/SwiftGDTests/ImageTests.swift:55]: :)

**Comment by twostraws** [comment]
This is great – thank you! Are you happy for me to go ahead and merge this now?

**Review by mcritz — APPROVED** [review]

**Pull request merged**